fault-readme.md

```diff
@@ -186,7 +186,7 @@ For example when you want to shift a bit mask, you can use the type "shift" as f
 The shift corresponds to a left shift. In the above case, the number 3 is subsequently shifted by 7, 8, 9 to the left (binary representation).
 
 Another example of a dictionary is the use of instruction width dependent fault masks.
-This only works in combination with the wildcard mode, the instruction fault type, and the overwrite fault model.
```

Comment:
I think it would be nice to add in addition a second example, that shows the config for a non-wildcard mode example.